LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 nameless-gos
+Copyright (c) 2018 disq-bio contributors
```

Comment:
Shouldn't it say "DISQ AUTHORS" instead? This repo has diferent authors than disq-bio organization, and with authors we pinpoint to the AUTHORS file.

Reply:
I think it is the project "Disq" (not it is *not* all caps).